packages/orchestration/README.md

```diff
@@ -326,6 +327,23 @@ const response = await orchestrationClient.chatCompletion({
 return response.getContent();
 ```
 
+### Using a JSON configuration from AI Launchpad
+
+If you already have a orchestration workflow created in AI Launchpad, you can copy or download the configuration as a JSON and use it directly with the orchestration client.
+
+```ts
+const jsonConfig = await fs.promises.readFile(
+  'path/to/orchestration-config.json',
+  'utf-8'
+);
+// Or alternatively, you can also provide the JSON configuration as a plain string in the code directly.
+// const jsonConfig = '... your JSON ...'
```

Comment:
Please refer to the [styleguide](https://github.com/SAP/cloud-sdk/blob/main/STYLEGUIDE.md#placeholders) for placeholders